From edfbeee75dcd03206a71764563a4ad868ef581a6 Mon Sep 17 00:00:00 2001
From: awakened <pham0050@e.ntu.edu.sg>
Date: Mon, 14 Nov 2022 17:36:48 +0100
Subject: [PATCH 19/19] SystemUI: make QS footer rounded (#910)

Current top left and top right of the QS footer are rounded, while bottom left and bottom right are not.
This commit makes all corners rounded, thus improving aesthetic appearance of the QS footer
---
 .../SystemUI/res/drawable/qs_footer_actions_background.xml    | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/packages/SystemUI/res/drawable/qs_footer_actions_background.xml b/packages/SystemUI/res/drawable/qs_footer_actions_background.xml
index a7e8762a2593..854efa73377f 100644
--- a/packages/SystemUI/res/drawable/qs_footer_actions_background.xml
+++ b/packages/SystemUI/res/drawable/qs_footer_actions_background.xml
@@ -17,6 +17,8 @@
     <shape>
         <solid android:color="?attr/underSurface"/>
         <corners android:topLeftRadius="@dimen/qs_corner_radius"
-                 android:topRightRadius="@dimen/qs_corner_radius"/>
+                 android:topRightRadius="@dimen/qs_corner_radius"
+                 android:bottomLeftRadius="@dimen/qs_corner_radius"
+                 android:bottomRightRadius="@dimen/qs_corner_radius"/>
     </shape>
 </inset>
-- 
2.25.1

